app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :photo, presence: true
+  validates :bio, presence: true
+  attr_accessor :posts_counter
+
+  has_many :posts, foreign_key: 'author_id'
+
```

Comment:
- [ ] As shown on the ERD diagram if you may look carefully, a User can have `many` comments and likes as well on top of the posts, So please make sure you specify that here in your user model as you did for posts.